dev_guide/managing_images.adoc

```diff
@@ -826,12 +826,6 @@ spec:
       scheduled: true
 ----
 
-[NOTE]
-====
-By default, `*importPolicy.scheduled*` is disabled and must be enabled in the
```

Comment:
Wait, there are two things mixed here. One is `importPolicy.scheduled` which is set *on an Image Stream*, per-tag to be honest. And second is the master config option `disableScheduledImport` which indeed is `false` by default. I'd suggest changing this description to explicitly mention that the IST's `scheduled` is disabled, and needs to be enabled explicitly.